src/main/java/org/StudentRepository/exceptions/InvalidStudentException.java

```diff
@@ -0,0 +1,8 @@
+package org.StudentRepository.exceptions;
+
+public class InvalidStudentException extends Exception {
```

Comment:
like we discussed during the course, checked exceptions should be avoided. Better to use unchecked exceptions

```suggestion
public class InvalidStudentException extends RuntimeException {
```